static/js/src/tests/intlTelInput.test.js

```diff
@@ -1,6 +1,6 @@
 import fetchMock from "jest-fetch-mock";
```

Comment:
shouldn't you rename this test file as well?

Reply:
good catch